### PR TITLE
Add storePreferencesData to orderForm query

### PR DESCRIPTION
**Pull request by gris**
#### What problem is this solving?

To be able to fetch `storePreferencesData` in the `orderForm` query. It is [available](https://github.com/vtex-apps/store-graphql/blob/master/graphql/types/OrderForm.graphql#L23) in store-graphql 

#### How should this be manually tested?

[Workspace](https://grispref--demostore.myvtex.com/_v/private/vtex.checkout-graphql@0.53.0/graphiql/v1?query=%7B%0A%20%20orderForm(orderFormId%3A%2214a7b62c9cb0490287396504b1390943%22)%7B%0A%20%20%20%20allowManualPrice%0A%20%20%20%20storePreferencesData%20%7B%0A%20%20%20%20%20%20countryCode%0A%20%20%20%20%20%20currencyCode%0A%20%20%20%20%20%20timeZone%0A%20%20%20%20%20%20currencySymbol%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D)

#### Checklist/Reminders

- [] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: node/yarn.lock
Comment (wdsrocha) [inline node/yarn.lock:1602]: Does it matter that this changed to npm?

Comment (lucasecdb) [reply node/yarn.lock:1602]: no

Comment (gris) [reply node/yarn.lock:1602]: Hum I'm not sure. Do you know how to fix it?

Comment (wdsrocha) [reply node/yarn.lock:1602]: I don't know how to fix this, but if it does not matter, it's fine by me.

**Review by wdsrocha — CHANGES_REQUESTED** [review]
LGTM, will approve once the conflicts from CHANGELOG have been resolved

**Comment by gris** [comment]
Done @wdsrocha !

**Review by wdsrocha — APPROVED** [review]
✨

**Comment by lucasecdb** [comment]
@gris typically we wait for two peer reviews before merging pull-requests

**Comment by gris** [comment]
Sorry! I didn't know about it 😭

**Comment by gris** [comment]
I will keep it in mind for next time. Is there something that you would like me to fix/revert?

**Comment by lucasecdb** [comment]
no, it's fine! 😄 it was just a heads-up, the config we added to this repo keeps reverting back to 1 review per PR :-(

**Comment by gris** [comment]
Should I deploy or you do?

**Comment by lucasecdb** [comment]
@gris go for it

**Pull request merged**
